src/pages/Apps/customApps/Electrs.tsx

```diff
@@ -47,95 +57,80 @@ const Electrs: FC = () => {
 
   if (appStatus.find((app) => app.id === "electrs")?.installed === false) {
     navigate("/apps");
-    return <></>;
-  }
-
-  const changeQR = (type: "local" | "tor") => {
-    switch (type) {
-      case "local":
-        setQRAddress(`${localIP}:${portSSL}:s`);
-        break;
-      case "tor":
-        setQRAddress(`${TORaddress}:${portSSL}:s`);
-        break;
-    }
-  };
-
-  if (!showTorConn && TORaddress && portSSL) {
-    setShowTorConn(true);
-  }
-
-  if (!showLocalConn && localIP && portSSL) {
-    setShowLocalConn(true);
+    return null;
   }
 
   return (
-    <main className="page-container content-container w-full bg-gray-700 text-white lg:pr-8">
+    <main className="page-container content-container w-full bg-gray-700 text-white flex flex-col items-center">
       {/* Back Button */}
       <section className="w-full px-5 py-9 text-gray-200">
-        <button
+        <Button
           onClick={() => navigate("/apps")}
-          className="flex items-center text-xl font-bold outline-none"
+          color="primary"
+          startContent={<ChevronLeftIcon className="inline-block h-5 w-5" />}
         >
-          <ChevronLeftIcon className="inline-block h-5 w-5" />
           {t("navigation.back")}
-        </button>
+        </Button>
       </section>
-      <section className="bd-card mx-5 flex flex-col gap-4 px-5 lg:flex-row">
-        <article className="lg:w-1/2">
-          {!initialSyncDone && <p>{t("appInfo.electrs.initialSync")}</p>}
-          {initialSyncDone && (
+
+      <Card className="bd-card">
+        <CardHeader>
+          <h3 className="text-xl">
```

Comment:
Wanna use `import { Headline } from "@/components/Headline";` for this?

This should work:
```
<Headline as="h3" size="xl">...
```

Reply:
Done